### DISCUSSION TITLE
Timescale for Liveliness

### DISCUSSION THREAD

**Discussion by smnrgrs** [eclipse-zenoh/roadmap#156] [Q&A] [answered]
I see that Liveliness is an unstable feature in the Rust implementation. Can anyone give me an idea of when it will be available in the C api?

**Comment by JEnoch** [comment, marked as answer]
Hi @smnrgrs ,
It's already available in zenoh-c ! But not yet in zenoh-pico.
- see here for 0.11.0 : https://zenoh-c.readthedocs.io/en/0.11.0/api.html#liveliness
- see here for 1.0.0-beta.1 : https://zenoh-c.readthedocs.io/en/1.0.0.7/api.html#liveliness

**Reply by smnrgrs** [reply]
Thanks for your response. I'll take another look!
